keps/sig-network/4444-service-traffic-distribution/README.md

```diff
@@ -387,7 +381,28 @@ traffic
   so as not to confuse with the actual process of selecting the complete set of
   pods backing a service.
 
-<<[/UNRESOLVED]>>
+### Renaming `PreferClose` to `PreferSameZone`
+
+This KEP previously proposed `PreferClose` as a possible value for the
+`trafficDistribution` field to provide flexibility for implementations to define
+"closeness." This open-ended approach aimed to accommodate both simple
+implementations (like kube-proxy, initially interpreting it as "prefer same
+zone") and more sophisticated ones (potentially offering "prefer same node with
+load-based fallback").
+
+However, this flexibility also introduced ambiguity. As discussions around
+adding a "prefer-same-node" option in
+[kubernetes/enhancements#4931](https://github.com/kubernetes/enhancements/pull/4931)
+illustrated, the lack of a precise definition for `PreferClose` raised concerns
+about overlapping behaviors and future extensibility. Having something like
+`PreferSameNode` alongside `PreferClose` could lead to confusion about the
+distinction.
+
+To pave the way for a future `PreferSameNode` option and to provide a more
+precise and less ambiguous meaning, `PreferClose` is being renamed
+`PreferSameZone`. (To ensure backward compatibility `PreferClose` can still be
+used as an alias for `PreferSameZone`.)
```

Comment:
This discussion is good, but it needs to be reworded since the rename isn't actually happening in this KEP.

And maybe move it up to the "Proposal" section when you explain what the semantics of `PreferClose` are? Or alternatively, move it down to the "Alternatives" section, and rewrite it along the lines of "Instead of having PreferClose mean something specific we could have had it mean something vague, but we didn't because ..."

Reply:
Thanks, the alternatives idea sounds good. I added an `An alternative definition of PreferClose` section with all relevant details and links.